### PR TITLE
Add python to CI

### PR DESCRIPTION
**Pull request by rasapala**

### CODE REVIEW

File: Dockerfile.openvino [hunk 1]
```diff
@@ -124,7 +124,7 @@ tar xvf boost_1_69_0.tar.gz && cd boost_1_69_0 && ./bootstrap.sh && \
 --with-log --with-locale \
 install
 
-ARG OVMS_BRANCH="mediapipe_integration"
+ARG OVMS_BRANCH="storage_lib_fix"
```
Comment (dtrawins) [inline Dockerfile.openvino]: can we switch to main?

Comment (rasapala) [reply Dockerfile.openvino]: Switching to main will introduce vulnerability for failures. I think we should switch to a release branch - after release, or use critical fixes branches - like in this example.

Comment (atobiszei) [reply Dockerfile.openvino]: We should declare specific sha of a commit from main here.

Comment (rasapala) [reply Dockerfile.openvino]: Done.

Comment (atobiszei) [reply Dockerfile.openvino]: I would assume that we should keep the same sha in WORKSPACE & Dockerfile - add comment in both places about that. +1 after adding comment.

**Review by atobiszei — APPROVED** [review]

**Review by dtrawins — APPROVED** [review]

**Pull request merged**